Makefile

```diff
@@ -63,6 +63,14 @@ deploy:
 	cd config/manager && kustomize edit set image controller=${IMG}
 	kustomize build config/default | kubectl apply -f -
 
+protoc-docker:
+	docker build - -t protoc < hack/grpc-protoc.Dockerfile
+
+protobuf: protoc-docker
+	docker run -v `pwd`:/eco -w /eco protoc:latest -I=api/proxy --go_out=plugins=grpc:api/proxy api/proxy/proxy.proto
+
+verify-protobuf:
+	./hack/verify.sh make -s protobuf
```

Comment:
How is the `make -s protobuf` arg(s) used? I don't see (m)any changes in the `verify.sh` script.

Reply:
`verify.sh` copies the code to a temporary directory, executes the given command (`make -s protobuf`), and then diffs for changes. If there are any it fails and tells you that you need to run the given command yourself. So it works without changes.